docs/schema-conversion-logic.md

```diff
@@ -0,0 +1,97 @@
+# Schema conversion logic for Whisper data
+
+Metrictank comes with schema conversion logic (`mdata/importer/converter.go`) which can be used to convert data from one storage schema into another one. This is currently only used by the Whisper importer, but may in the future also be used to import from other types of data sources.
```

Comment:
Maybe move this first line up above?
Something like:

#Schema conversion
intro

## Whisper

Reply:
not sure if it's necessary/a good idea, just feels like there's a bit of a mismatch between the file name (generalized schema conversion) and the content/header (just whisper, which kind of makes sense since that's all we can do at the moment)

Reply:
most of the schema conversion logic is actually supposed to be generic, so it doesn't make sense to put it all under a `Whisper` title. So how about I just rename the top title from `Schema conversion logic for Whisper data` to `Schema conversion`

Reply:
:shipit: